.code-samples.meilisearch.yaml

```diff
@@ -354,7 +351,7 @@ faceted_search_facets_distribution_1: |-
     })
 faceted_search_walkthrough_attributes_for_faceting_1: |-
```

Comment:
The name of this ID is not accurate anymore, it should be changed depending on the title of the section or the page.